### DISCUSSION TITLE
Plot_model

### DISCUSSION THREAD

**Discussion by carlosaur** [pycaret/pycaret#1171] [Q&A] [answered]
I made a clustering model with Kmeans. 
When I try to use "plot_model" for "distribution" or "cluster" types, an error message appears: "PandasArray must be 1-dimensional".
The original dataframe has more than 60 columns.
Does anyone have any idea what might be happening?

**Comment by Yard1** [comment]
Can you provide a reproducible example in Google Colab?

**Comment by carlosaur** [comment, marked as answer]
I did a new test and it worked this time.
Maybe it has to do with the amount of RAM available on my laptop at the time of testing.
I will continue testing and if it fails, I will post again.
thanks.
